src/gui/screen_home.cpp

```diff
@@ -232,7 +244,8 @@ void screen_home_data_t::printBtnDis() {
     w_labels[0].SetText(_(labels[labelNoUSBId]));
 }
 
-bool screen_home_data_t::moreGcodesUploaded() {
+bool screen_home_data_t::MoreGcodesUploaded() {
```

Comment:
This capitalization change goes against the style guideline we agreed on. Is there a good reason for it?